<launch>
    <node name="aruco_estimator" pkg="uav_landing" type="aruco_estimator" respawn="false" output="screen">
        <rosparam file="$(find uav_landing)/config/positioning/aruco_estimator.yaml" command="load"/>
    </node>

</launch>